R/sim_gs_n.R

```diff
@@ -223,8 +215,7 @@
 #'   enroll_rate = enroll_rate,
 #'   fail_rate = fail_rate,
 #'   test = list(ia1 = ia1_test, ia2 = ia2_test, fa = fa_test),
-#'   cut = list(ia1 = ia1_cut, ia2 = ia2_cut, fa = fa_cut),
-#'   seed = 2024
+#'   cut = list(ia1 = ia1_cut, ia2 = ia2_cut, fa = fa_cut)
 #' )
 #' }
```

Comment:
Similar to `sim_fixed_n()`, we should update one of the examples to use `plan("multisession", workers = 2)`

https://github.com/Merck/simtrial/blob/f8a16eea5ca1ba5ba8b08128c91b64e9c5f63061/R/sim_fixed_n.R#L110